consensus/types/src/eth_spec.rs

```diff
@@ -296,6 +297,10 @@ pub trait EthSpec:
         Self::DataColumnCount::to_usize()
     }
 
+    fn min_custody_requirement() -> usize {
```

Comment:
I think this is the same as [`spec.custody_requirement`](https://github.com/sigp/lighthouse/blob/ae470d70041339f00de5d404ff75bcc3e0cf31eb/consensus/types/src/chain_spec.rs#L698)?

Reply:
yea, looks like it! is there a reason to prefer `ChainSpec` over `EthSpec`? it's not immediately clear what the overlap is between the two. at least from a typing perspective, it seems that the use of `EthSpec` is more ubiquitous.

Reply:
Yeah good question and I don't know if I have the right answer and kinda assumed it for a while! lol

I think we use `EthSpec` for config that we need to know at compile time (and we use num types), and `ChainSpec` contains preset values loaded from files at runtime. I'll confirm with the team to make sure I'm not giving false answers though :)

Reply:
fwiw it helps to have the custody requirement available in `EthSpec`, since the methods on the `Eth2Enr` trait have a generic parameter for `EthSpec`.